### ISSUE TITLE
Add Mandrill sending integration w/auto update of subscriptions based on Mandrill data

### ISSUE THREAD

**Issue by tevyaw** [websharks/comment-mail#217]
Add deeper Mandrill integration, including sending with it. But them make it so Comment Mail can remove subscriptions that repeatedly fail to deliver, and immediately remove any that mark the message as spam. This way we keep our Mandrill ratings high, since we're not repeatedly sending to addresses that don't work, or to people who are marking our messages as spam (even though they signed up for them).

**Comment by jaswsinc** [comment]
> Add deeper Mandrill integration, including sending with it.

You can use Mandrill as an SMTP provider if you have Comment Mail Pro.
See: **Dashboard → Comment Mail → Config. Options → SMTP Server Integration**

> Comment Mail can remove subscriptions that repeatedly fail to deliver, and immediately remove any that mark the message as spam.

I agree. cc @raamdev This seems like an important component that we are missing currently. It would be great if Comment Mail had hooks that receive details about delivery failures and spam complains from Mandrill, SendGrid, SES, etc. If we had those in place, a site owner could then configure rules that would cause emails to be purged automatically after X number of delivery failures.

Referencing: https://mandrill.zendesk.com/hc/en-us/articles/205583467-High-Bounce-Rate
See also: https://mandrill.zendesk.com/hc/en-us/articles/205583217-Introduction-to-Webhooks

---

Short of this, we could try to research services that deal with this issue specifically; i.e., instead of putting this into Comment Mail itself, write KBAs that show site owners various ways to deal with this.

**Comment by tevyaw** [comment]
Yeah, I know you can send with it. I was thinking an API level integration that would both handle the sending (no SMTP config required) and be able to hook in to the data. The rules you describe sound perfect.

BTW: this one is much higher on my wish-list than my other suggestion about replying to MailPoet emails to comment.
